geopandas/base.py

```diff
@@ -2136,6 +2136,14 @@ def union_all(self, method="unary"):
               unary union algorithm. However, it can produce invalid geometries if the
               polygons overlap.
 
+        grid_size : float, default None
```

Comment:
Can we also add a sentence why this could be useful?

Reply:
I gave a try to add this... and tried to make the description less technical...